gdcdictionary/schemas/sample.yaml

```diff
@@ -405,7 +408,6 @@ properties:
       - "No"
       - Unknown
       - Not Reported
-      - Not Allowed To Collect
   freezing_method:
```

Comment:
Remove? Should be covered by `preservation_method`?

Reply:
Removed `freezing_method`